test/CodeGenILTests/RangeMicroTests.cs

```diff
@@ -11,21 +11,39 @@ namespace CodeGenILTests
 {
     public class MicroTests
     {
+        private ProtoScript.Runners.LiveRunner liveRunner = null;
+        private ProtoCore.MSILRuntimeCore runtimeCore = null;
+
         protected EmitMSIL.CodeGenIL codeGen;
         protected Dictionary<string, IList> inputs = new Dictionary<string, IList>();
 
+        protected virtual void GetLibrariesToPreload(ref List<string> libraries)
+        {
+            libraries.Add("DesignScriptBuiltin.dll");
+            libraries.Add("DSCoreNodes.dll");
+        }
+
         [SetUp]
         public void Setup()
         {
+            liveRunner = new ProtoScript.Runners.LiveRunner();
+
+            List<string> libraries = new List<string>();
+            GetLibrariesToPreload(ref libraries);
+
+            liveRunner.ResetVMAndResyncGraph(libraries);
+            runtimeCore = new ProtoCore.MSILRuntimeCore(liveRunner.RuntimeCore);
```

Comment:
Is this a good long-term strategy to establish reliance on the old VM? It looks like it will be harder to split away from it later when we want to ditch it completely or ditch parts of it.

Reply:
Well...if we do go with the new VM,  we will have to implement the EmitFunctionDefinition and EmitImportStatements and all the preloading logic.
This is a temporary solution, a shortcut to get access to all the loaded types

Reply:
can we file a task to implement `EmitFunctionDefinition` and `EmitImportStatements`, add a TODO to the code, and replace these calls?

Reply:
will do